Source/App/AppController.swift

```diff
@@ -26,6 +26,15 @@ class AppController: UIViewController {
     private var didStartDatabaseProcessingObserver: NSObjectProtocol?
     private var didFinishDatabaseProcessingObserver: NSObjectProtocol?
     private var didUpdateDatabaseProgressObserver: NSObjectProtocol?
+    
+    /// One and a half second `CATransition` `.easeInEaseOut` with type `CATransitionType.fade`.
+    private var crossFadeTransition: CATransition {
+        let transition = CATransition()
+        transition.duration = 1.5
```

Comment:
This animation felt pretty slow to me. I think Apple's transitions are usually about 0.3 seconds, and most of the ones in Planetary are around that duration as well. Are you ok with matching that?

Reply:
Fixed.